README.md

```diff
@@ -28,16 +28,16 @@ This node SDK is designed to help developers easily integrate Onqlave `Encryptio
 - Encrypt/Decrypt stream of data
 
 ## Installation
+```sh
+npm install @onqlavelabs/onqlave-node
 
+```
 ### Requirements
 - Node 7.6.0 and above
```

Comment:
Can we change this to Node 16.0.0 and above?

Reply:
updated